workspace/apps/pidp/src/app/core/services/utils.service.ts

```diff
@@ -3,6 +3,8 @@ import { Inject, Injectable } from '@angular/core';
 
 import { delay, of } from 'rxjs';
 
+import * as CryptoJS from 'crypto-js';
```

Comment:
I think we can add these crypto related functions into a file called crypto.service.
So that if any additional functions comes in future we can place them in that file.
It will more readable and easy to navigate.